ultra/tests/test_env.py

```diff
@@ -61,6 +63,22 @@ def run_experiment():
         self.assertTrue(task_id1 == task_id)
         self.assertTrue(task_level1 == task_level)
 
+    def test_scenario_sequence(self):
+        @ray.remote(max_calls=1, num_gpus=0, num_cpus=1)
+        def run_experiment():
+            _, env = prepare_test_env_agent(
+                scenario_info=[prebuilt_scenario1, prebuilt_scenario2]
+            )
```

Comment:
Do we need to initialize an environment to retrieve the scenarios? I am wondering that the `get_scenarios_from_scenario_info` is a static method, then why don't we just test that method without initializing an env. This way we can ensure that the `get_scenarios_from_scenario_info` works from  a static method POV

Reply:
If we include the static method test, should we get rid of the `test_scenario_sequence` test?

Reply:
Yeah, I do not think the `test_scenario_sequence` test is needed